RTC::LogicalTimeTriggeredEC::svc のスリープ時間のログ出力処理の確認

<!--
***********************************************************************
*                           << NOTE >>                                *
*  This repository have been programmed in C++.                       *
*                                                                     *
*  Please do not create tickets for other programming languages here. *
*  If you will create tickets for other programming language,         *
*  please see below:                                                  *
*                                                                     *
*  - Python : https://github.com/OpenRTM/OpenRTM-aist-Python/issues   *
*  - Java   : https://github.com/OpenRTM/OpenRTM-aist-Java/issues     *
*                                                                     *
***********************************************************************
-->

コンパイラの警告対応中に下記の警告が出力されました。
この警告箇所の変数が必要ではないかと思ったので確認させてください。

#483 にて変更されています。
指摘箇所の後にある処理の RTC_PARANOID は t3 -t0 ではなく t2 からのスリープ時間で t3-t2 のtypo？

~~~
/root/OpenRTM-aist/src/ext/ec/logical_time/LogicalTimeTriggeredEC.cpp: In member function 'virtual int RTC::LogicalTimeTriggeredEC::svc()':
/root/OpenRTM-aist/src/ext/ec/logical_time/LogicalTimeTriggeredEC.cpp:152:14: warning: variable 't2' set but not used [-Wunused-but-set-variable]
         auto t2 = std::chrono::steady_clock::now();
~~~

--- a/src/ext/ec/logical_time/LogicalTimeTriggeredEC.cpp
+++ b/src/ext/ec/logical_time/LogicalTimeTriggeredEC.cpp
@@ -158,7 +158,7 @@ namespace RTC
         if (true) //count > 1000)
           {
             auto t3 = std::chrono::steady_clock::now();
-            RTC_PARANOID(("Slept:       %f [s]", std::chrono::duration<double>(t3 - t0).count()));
+            RTC_PARANOID(("Slept:       %f [s]", std::chrono::duration<double>(t3 - t2).count()));
             count = 0;
           }
         ++count;

